lib/i18n/backend/simple.rb

```diff
@@ -29,6 +29,11 @@ def initialized?
         # translations will be overwritten by new ones only at the deepest
         # level of the hash.
         def store_translations(locale, data, options = {})
+          if I18n.available_locales_initialized? &&
+            I18n.available_locales.include?(locale.to_sym) == false &&
```

Comment:
Please use `!I18n.available_locales.include?(locale.to_sym)` here instead of `== false`.